.github/workflows/build.yml

```diff
@@ -219,6 +219,10 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
```

Comment:
should be in build-docs, not in test-windows

Reply:
Yep ... fixed !